com.cocoatech.deckr.yaml

```diff
@@ -0,0 +1,129 @@
+app-id: com.cocoatech.deckr
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: deckr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=home
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+modules:
+  - name: deckr
+    buildsystem: simple
+    only-arches:
+      - x86_64
```

Comment:
this doesn't prevent an aarch64 build.

Reply:
I have no idea what that does, I copied it from somewhere.  What should it be?